tests/api/runtimes/test_nuclio.py

```diff
@@ -52,8 +54,6 @@ def custom_setup(self):
         os.environ["V3IO_USERNAME"] = self.v3io_user = "test-user"
 
     def _serialize_and_deploy_nuclio_function(self, function):
-        # simulating sending to API - serialization through dict
-        function = function.from_dict(function.to_dict())
         deploy_nuclio_function(function)
```

Comment:
What left of `_serialize_and_deploy_nuclio_function` is exactly the same as `_execute_run` so I guess remove it, and use `_execute_run` everywhere